web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx

```diff
@@ -132,7 +132,15 @@ export const ChangesetNode: React.FunctionComponent<ChangesetNodeProps> = ({
                                 )}
                                 {node.labels.length > 0 && (
                                     <span className="ml-2">
-                                        {node.labels.map(label => (
+                                        {node.labels.sort((a,b)=> {
+                                            if (a.text > b.text) {
+                                                return 1;
+                                            }
+                                            if (a.text < b.text) {
+                                                return -1;
+                                            }
+                                            return 0;
+                                        }).map(label => (
```

Comment:
lodash can do this :)
needs `import { sortBy } from 'lodash'`
```suggestion
                                        {sortBy(node.labels, 'text').map(label => (
```